frontend/ui/otamanager.lua

```diff
@@ -222,6 +222,8 @@ function OTAManager:fetchAndProcessUpdate()
                 ok_callback = function()
                     if Device:isSDL() then
                         os.execute("xdg-open '"..link.."'")
+                    elseif Device:isAndroid() then
+                        Device:update(link, "koreader-lastest.apk")
```

Comment:
I don't understand the rationale behind wanting to discard the perfectly nice filename for one lacking in clarity.

(As an aside, it'd be `latest`.)

Reply:
I was thinking in "koreader-2019-xx-xxx.apk"

Reply:
Definitely not the place of this PR. That falls under #4297.

Reply:
See below